flyteidl/protos/flyteidl/core/literals.proto

```diff
@@ -140,6 +143,18 @@ message LiteralMap {
     map<string, Literal> literals = 1;
 }
 
+// A collection of fields for tuple literal.
+message LiteralTupleMap {
+    // The name of the NamedTuple. If it is original tuple, it would be empty string.
+    string tuple_name = 1;
+
+    // The order of each fields stored in the tuple.
+    repeated string order = 2;
+    
+    // A map of literals.
+    map<string, Literal> literals = 3;
```

Comment:
Can we use `Literal.collection`?

Reply:
`Literal.collection` is not enough to store the name of the fields for `NamedTuple`.